addons/website/static/tests/tours/edit_translated_page.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('website.edit_translated_page', function (require) {
+'use strict';
+
+const tour = require("web_tour.tour");
+
+tour.register('edit_translated_page_redirect', {
+    test: true,
+    url: '/fr_FR',
```

Comment:
:grimacing: :thinking:

Reply:
Byt he way, This URL is used when you run the tour in your browser directly (`odoo.startTour("xxx")`)